app/assets/javascripts/action_plan.js

```diff
@@ -0,0 +1,9 @@
+document.addEventListener("turbolinks:load", function() {
+  $("input[name='action-plan_prefix']").keyup(function(event) {
```

Comment:
If you give the input an id, then you can use the id over here
```suggestion
  $("#action-plan_prefix").keyup(function(event) {
```